enf/provider.go

```diff
@@ -51,3 +56,15 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 
 	return config.Client()
 }
+
+func validateRateLimitRequestWhenInheriting(d *schema.ResourceData) error {
```

Comment:
The calling code would be simpler if this function checked both `inheriting and (not ratelimits set)` rather than just `not ratelimit set`. 

E.g.,
1) rename to `validateRateLimitRequest`
1) return `nil` if inherit is false, otherwise check if the packet rates are set
1) change the callers from
```go
if d.Get("inherit") == true {
  if err := validateRateLimitRequestWhenInheriting(d); err != nil {
    return err
  }
}
```
to
```go
if err := validateRateLimitRequest(d); err != nil {
  return err
}
```

Reply:
I like this change. Good suggestion!